### PR TITLE
SPM support documentation

### PR DESCRIPTION
**Pull request by RomanPodymov**
Hello.
Thank you for CoreLocation.
Add SPM support documentation.

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -5,15 +5,23 @@ This project adds promises to Apple’s CoreLocation framework.
 ## CocoaPods
 
 ```ruby
-pod "PromiseKit/CoreLocation", "~> 6.0"
+pod "PromiseKit/CoreLocation", "~> 3.1.2"
```
Comment (mxcl) [inline README.markdown]: this is incorrect I believe. CoreLocation is a subspec and thus the version is tied to the PromiseKit version.

File: README.markdown [hunk 2]
```diff
@@ -5,15 +5,23 @@ This project adds promises to Apple’s CoreLocation framework.
...
 ## Carthage
 
 ```ruby
-github "PromiseKit/CoreLocation" ~> 3.0
+github "PromiseKit/CoreLocation" ~> 3.1.2
+```
+
+## Swift Package Manager
+
+```ruby
```
Comment (mxcl) [inline README.markdown]: s/swift/ruby/

File: README.markdown (hunk 1 above)
Comment (RomanPodymov) [reply README.markdown]: Reverted.

File: README.markdown (hunk 2 above)
Comment (RomanPodymov) [reply README.markdown]: Fixed.

**Review by mxcl — APPROVED** [review]

**Pull request merged**